opm/autodiff/StandardWellsDense.hpp

```diff
@@ -106,6 +114,10 @@ namespace Opm {
                     return;
                 }
 
+                // assumes the gas fractions are stored after water fractions
+                // WellVariablePositions needs to be changed for 2p runs
+                assert (np == 3 || np == 2 && !pu.phase_used[Gas] );
```

Comment:
This breaks debug builds (or, more specifically, builds for which `NDEBUG` is **not** defined).  The assert refers to objects `np` and `pu`, none of which exist at this point.  Please fix.